pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -210,6 +210,15 @@ func TestUpdateStatus(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 	pvOut := obj.(*api.PersistentVolume)
+
+	if pvStartTimestamp.Equal(pvOut.Status.LastPhaseTransitionTime) {
+		t.Errorf("unexpected timestamp, expected %v, got %v", pvOut.Status.LastPhaseTransitionTime, pvStartTimestamp)
+	}
+
+	// reset timestamp to nil in order to compare the rest of the object
```

Comment:
Do we know how the drift was caused making the DeepEqual fail? I don't see it yet but removing the root cause would be better than trying to mask it.

Reply:
the root cause is that the timestamp is updated due to the phase changed from pending.

the previous test is not correct even though its result is passed. If the execution process of test is beyond a certain threshold, it will fail.

Reply:
Agree, the timestamp really updates due to phase change in this case but that's what I wanted to avoid. See the comment below.